testing/nicotine-plus/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer:
+pkgname=nicotine+
```

Comment:
pkgname must be the same as name of the abuild's directory and must not contain `+`.